index.d.ts

```diff
@@ -82,13 +82,29 @@ declare module '@silintl/ui-components' {
     }
   }
 
+  interface DateInputProps {
```

Comment:
Do you want to add `class`, since it is a prop that can be provided and will be used (even though it's not `export`ed the way normal props are)?

Reply:
Good question. It seems we must be getting class from somewhere already as I don't get and error from using `class` when its not declared in this file. I might still be good to include anyways.